packages/datetime/src/dateInput.tsx

```diff
@@ -182,13 +190,14 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
         return (
             <Popover
                 autoFocus={false}
-                content={popoverContent}
                 enforceFocus={false}
```

Comment:
A couple of these should _not_ be overridable based on my experience with https://github.com/palantir/blueprint/pull/940, particularly `autoFocus` and `enforceFocus`. Led to really weird UX.